src/winmain.cpp

```diff
@@ -1085,7 +1088,12 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR lpszCmdLine, int)
 				string noUpdate = nativeLang.getMessageString("MSGID_NOUPDATE");
 				if (noUpdate == "")
 					noUpdate = MSGID_NOUPDATE;
-				::MessageBoxA(NULL, noUpdate.c_str(), gupParams.getMessageBoxTitle().c_str(), MB_OK);
+				int res = ::MessageBoxA(NULL, noUpdate.c_str(), gupParams.getMessageBoxTitle().c_str(), MB_OKCANCEL);
+				if (res == IDOK)
+				{
+					// This URL is better to be in config
+					::ShellExecute(NULL, TEXT("open"), TEXT("https://notepad-plus-plus.org/downloads/"), NULL, NULL, SW_SHOWNORMAL);

```

Comment:
Didn't find a good place so it is here for now.
It could be moved to XML config or passed as a command-line argument.